src/MiddlewareInterface.php

```diff
@@ -9,7 +9,7 @@
 {
     /**
      * Process an incoming server request and return a response, optionally delegating
-     * to the next middleware component to create the response.
+     * to the next request processor to create the response.
```

Comment:
The documentation for the PSR already uses the term "middleware component" so changing this to "request processor" would introduce a term not used anywhere else.